subsys/bluetooth/services/fast_pair/fmdn/Kconfig

```diff
@@ -323,6 +323,15 @@ config BT_FAST_PAIR_FMDN_DULT_FIRMWARE_VERSION_REVISION
 
 endmenu
 
+config BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR
+	bool "Enable DULT Motion detector support in FMDN"
+	default y
```

Comment:
let's keep it disable by default as this optional feature of the FMDN/DULT (similarly to the BT_FAST_PAIR_FMDN_BATTERY_DULT). We can simply enable it in the prj.conf of the locator tag sample.